source/pkcore/abilities.rst

```diff
@@ -66,14 +63,12 @@ Waterbending abilities focus on being able to adapt between offense, defense, an
 Chiblocking
 ===========
 Chiblocking is the anti-bending fighting style. Chiblockers use critically
-placed kicks and punches to block chi paths in bender's bodies, doing
-anything from disabling their bending to temporarily paralyzing
-them.
+placed kicks and punches to block chi paths in bender's bodies disabling their bending.
 
-Chiblocking abilities are purely for combat, being able to temporarily block a bender's chi so they are unable to bend.
+Chiblocking abilities focus on close quarters combat countering bending abilities.
```

Comment:
> Chiblocking abilities focus on close quarters combat and countering bending abilities.

Reply:
Chiblockers deal a lot of damage melee regardless of whether the target is a bender or not. I think that having that separation is important.

Reply:
oop typo